File: Qubit Server/dev/src/main/java/org/labrad/qubits/FpgaModel.java
package org.labrad.qubits;

import org.labrad.qubits.resources.DacBoard;

public interface FpgaModel {

  public String getName();
  public DacBoard getDacBoard();
  public double getSequenceLength_us();
  public double getSequenceLengthPostSRAM_us();
}

File: QubitServer/src/main/java/org/labrad/qubits/FpgaModel.scala
package org.labrad.qubits

import org.labrad.qubits.resources.DacBoard

trait FpgaModel {

  def getName(): String
  def getDacBoard(): DacBoard
  def getSequenceLength_us(): Double
  def getSequenceLengthPostSRAM_us(): Double
}
